components/omega/src/infra/logging.cpp

```diff
@@ -0,0 +1,13 @@
+// Omega logging implementation file.
```

Comment:
We are following llvm style ( https://llvm.org/docs/CodingStandards.html#introduction ) so so start filename with capital letter.  And the first line of the file should have a line as shown in LLVM style (or look at current files for DataTypes and MachEnv for examples).

Reply:
OK